apps/designer/app/designer/designer.tsx

```diff
@@ -37,6 +37,7 @@ import { usePublishShortcuts } from "./shared/shortcuts";
 import { type SyncStatus } from "~/shared/sync";
 import { useIsPreviewMode, useRootInstance } from "~/shared/nano-states";
 import { useSubscribeClientSetting } from "./shared/client-settings";
+import { Asset } from "@prisma/client";
```

Comment:
we don't import prisma directly anywhere, to have a single source of truth for data related stuff, we import from webstudio-react-sdk and will move data stuff into a separate package soon, but for now its there

Reply:
Will fix that then

Reply:
Fixed